zilencer/management/commands/populate_db.py

```diff
@@ -467,6 +467,22 @@ def handle(self, *args: Any, **options: Any) -> None:
 
             create_users(zulip_realm, names, tos_version=settings.TERMS_OF_SERVICE_VERSION)
 
+            # Add time zones to some users
```

Comment:
If we did timezone assignment a bit later in the outer function here, we could take advantage of having local vars for `iago`, `desdemona`, `shiva`, 'polonius`, etc.  I think it would make the code a bit more concise and save a round trip.

Reply:
I think the right actual fix would be to modify the `bulk_create_users` API to replace the current `(name, email)` tuple data structure with a set of objects where we could arrange things like having the `is_staff` flag.  I'm comfortable merging this with a TODO comment, since that refactor would be reasonably involved.